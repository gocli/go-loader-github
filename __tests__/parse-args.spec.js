const parseArgs = require('../lib/parse-args')

describe('Parse Arguments', () => {
  it('handles empty list without fail', () => {
    expect(parseArgs()).toMatchSnapshot()
  })

  it('handles install flag', () => {
    expect(parseArgs(['cmd', '--install'])).toMatchSnapshot()
  })

  it('handles no-install flag', () => {
    expect(parseArgs(['cmd', '--no-install'])).toMatchSnapshot()
  })
})
