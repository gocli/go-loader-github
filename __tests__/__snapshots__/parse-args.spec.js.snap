// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parse Arguments handles empty list without fail 1`] = `
Object {
  "_": Array [],
  "install": true,
}
`;

exports[`Parse Arguments handles install flag 1`] = `
Object {
  "_": Array [
    "cmd",
  ],
  "install": true,
}
`;

exports[`Parse Arguments handles no-install flag 1`] = `
Object {
  "_": Array [
    "cmd",
  ],
  "install": false,
}
`;
